cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -155,9 +155,7 @@ func getShardIDList(cmd *cobra.Command) [][]byte {
 	}
 
 	sidList, _ := cmd.Flags().GetStringSlice(shardIDFlag)
-	if len(sidList) == 0 {
-		common.ExitOnErr(cmd, "", fmt.Errorf("either --%s or --%s flag must be provided", shardIDFlag, shardAllFlag))
-	}
+	// guaranteed by MarkFlagsMutuallyExclusive
```

Comment:
Not sure it's obvious what's guaranteed here without this diff.

Reply:
agree, explained